MonkeyWrench.Web.WebService/WebServices.asmx.cs

```diff
@@ -3098,6 +3393,26 @@ public WebServiceResponse AddLaneNotification (WebServiceLogin login, int lane_i
 		}
 
 		[WebMethod]
+		public WebServiceResponse RemoveLaneNotificationForLane (WebServiceLogin login, int id, int lane_id)
+		{
+			WebServiceResponse response = new WebServiceResponse ();
+
+			using (DB db = new DB ()) {
+				var lane = DBLane_Extensions.Create (db, lane_id);
+				VerifyUserInRoles (db, login, lane.additional_roles, false);
+
+				using (IDbCommand cmd = db.CreateCommand ()) {
+					cmd.CommandText = "DELETE FROM LaneNotification WHERE id = @id;";
```

Comment:
You need to verify that the notification is from lane_id (should be just a matter of adding the condition to the sql statement).